src/devices/Fsr408/Fsr408.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+
+namespace Iot.Device.Fsr408
+{
+    public class Fsr408 : IDisposable
+    {
+        private int _pinNumber = 0;
+
+        private Mcp3008.Mcp3008 _adcConverter;
+        public int Resistance { get; set; } = 10000; // 10k ohm
+        public int PowerSupplied { get; set; } = 5000;  // 5 mV
+
+        public Fsr408(Mcp3008.Mcp3008 adcConverter)
+        {
+            _adcConverter = adcConverter;
+        }
+
+        public Fsr408(int pinNumber, Mcp3008.Mcp3008 adcConverter)
+        {
+            _adcConverter = adcConverter;
+            _pinNumber = pinNumber;
+        }
+
+        public int Read()
```

Comment:
What exactly is this reading? Voltage, Resistance, Force? Not sure what this method is supposed to do. As an aside, could you please add summary comments for all of the public sruface API?